pkg/package-server/server/server.go

```diff
@@ -82,7 +97,7 @@ func NewPackageServerOptions(out, errOut io.Writer) *PackageServerOptions {
 		Authorization:  genericoptions.NewDelegatingAuthorizationOptions(),
 		Features:       genericoptions.NewFeatureOptions(),
 
-		WakeupInterval: 5 * time.Minute,
+		WakeupInterval: DefaultWakeupInterval,
```

Comment:
In order to avoid an entire restart, could we change the `WakeupInterval` field to a `WakeupIntervalFunc` function. We could call the function every time we need the value, and we could have that function interrogate our OLMConfig cache to get the up-to-date value.

Reply:
The only time the value matters is when we pass it into:
```
sourceProvider, err := provider.NewRegistryProvider(ctx, crClient, queueOperator, o.WakeupInterval, o.GlobalNamespace)
```
Not sure how a function would be of benefit here?

Reply:
Ah, well we'd have to plumb the function through to the place where the wakeup interval is actually used.

Reply:
Yup, it's the plumbing that would need to be introduced, and eventually into k8s code(!). I gave up looking at it when the value reached the k8s code.